### PR TITLE
Emit in bulk in syncResolved

### PR DESCRIPTION
**Pull request by suzaku**
### What problem does this PR solve? 

We didn't utilize the bulk interface of Sink.

### What is changed and how it works?

Emit in bulk instead of saving txns one by one.

### Check List 

Tests 

 - Unit test

### CODE REVIEW

**Comment by amyangfei** [comment]
The coverage stage failure is caused by incompleted coverage information, it seems we haven't implemented the signal handling, I will file another PR to solve this problem. Just re-run CI and have a try, or ignore the error in this PR.

File: cdc/processor.go [hunk 1]
```diff
@@ -539,35 +552,53 @@ func (p *processor) syncResolved(ctx context.Context) error {
...
+			}
+
+			if err := p.schemaStorage.HandlePreviousDDLJobIfNeed(rawTxn.Ts); err != nil {
+				return errors.Trace(err)
+			}
+			txn, err := p.mounter.Mount(rawTxn)
+			if err != nil {
+				return errors.Trace(err)
+			}
+			pendingTxns = append(pendingTxns, txn)
```
Comment (leoppro) [inline cdc/processor.go]: I think we should filter the fake txn here.

**Comment by leoppro** [comment]
rest LGTM

File: cdc/processor.go (hunk 1 above)
Comment (suzaku) [reply cdc/processor.go]: Isn't fake txns already filtered in the `ddlJobsCh` case?

**Comment by leoppro** [comment]
LGTM

**Pull request merged**